src/main/java/com/spring/amazon/mapper/UserMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.spring.amazon.mapper;
+
+import com.spring.amazon.dto.ReviewDto;
+import com.spring.amazon.model.Role;
+import com.spring.amazon.model.User;
+import java.util.Set;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
+    private static final String PASSWORD = "1111";
+    private static final String ROLE_NAME = "USER";
+
+    public User mapToUser(ReviewDto reviewDto) {
+        User user = new User();
+        user.setExternalId(reviewDto.getUserId());
+        user.setProfileName(reviewDto.getProfileName());
+        user.setPassword(PASSWORD);
+        user.setRoles(Set.of(Role.of(ROLE_NAME)));
```

Comment:
we should create a separate class in our project where we add all the entities that were parsed from the file to the database

in this class we should fetch the role by rolename from the database (it should have id), and then set this role to the user

so, let's create that class and remove setting the roles and password from here